mizar/daemon/interface_service.py

```diff
@@ -33,6 +33,7 @@ def __init__(self):
         self.interfaces_q = queue.Queue()  # Used for Produce/Consume sync
         self.iproute = IPRoute()
         self.interfaces = {}  # In-memory tracking for Pod interfaces
+        self.known_substrates = {}
```

Comment:
We don't need this, it can be removed.
The transit xdp program does not need to update/delete the substrate information about its own node.